cmd/res/profiles/control-plane.profile.yaml

```diff
@@ -0,0 +1,29 @@
+name: "device-onvif-camera"
+manufacturer:  "Intel"
+model: "Control Plane Device for device-onvif-camera device service"
+labels:
+  - "onvif"
+  - "control-plane"
+description: "EdgeX device profile for device-onvif-camera Control Plane Events."
+
+deviceResources:
+  - name: "CameraAdded"
+    isHidden: true
```

Comment:
I like to put a comment on this line for each resource stating that the resource is Async only, co no commands.

Reply:
@brianointel here is an example:
https://github.com/marcpfuller/device-rfid-llrp-go/blob/7b55924c09c2d909d14f4730cd7f1b6443ec55e3/cmd/res/profiles/llrp.device.profile.yaml#L64